panacea/src/pml_parser.yrl

```diff
@@ -169,6 +191,45 @@ operator -> '>=' : ">=".
 
 Erlang code.
 
+validate_time({TimeType, [{line, Line}], Val}) ->
```

Comment:
I guess even a function to wrap the return error with the values like `time_out_of_range_error(TimeType, MaxVal, ActualVal)` would be better than whats there now.